README.md

```diff
@@ -451,17 +452,19 @@ In general, startups rely on investors to help fund rapid growth.
 🄳 In order to finance building or scaling their business, startups **fundraise** by selling
 shares in their business to [investors](http://www.investopedia.com/terms/i/investor.asp) in
 exchange for capital.
-Startups that aspire to grow rapidly are likely to fundraise from
-[**venture capital**](https://en.wikipedia.org/wiki/Venture_capital) investors, who are
-individuals or firms that specialize in such investments.
-After a company IPOs and is public, it can seek investment in public markets.
+Startups that aspire to grow rapidly are likely to fundraise from individuals or firms
+specializing in startup investment, called
+[**venture capital**](https://en.wikipedia.org/wiki/Venture_capital). After a company goes public
+in what is known as an [initial public offering (IPO)](#ipos), it can seek investment in
```

Comment:
Stylistically, this is in a definition so sounds like we're defining a second term. Let's omit that language but include the link.

Reply:
Fixed. Kept "goes public" in place of "IPOs and is public" for clarity.